### DISCUSSION TITLE
Help with augmented-ui and inputs

### DISCUSSION THREAD

**Discussion by Dominus98** [propjockey/augmented-ui#17] [General]
Hello, I was trying to create a login form using this wonderful style but at the moment I start stylizing the input for username and password, I don't get the borders for the clip and even the border don't use the color I want.
Maybe augmented-ui can't work with form inputs and I didn't read it.

You can check it right here: https://codepen.io/Dominus98/pen/yLZYzbG

Thank you in advance!

**Comment by JaneOri** [comment]
Hi Dominus98!
Thank you very much for checking it out, glad you're enjoying it!

Input elements don't allow ::before and ::after pseudo els so you've just gotta wrap the input - wish I could change that but hopefully that'll work for you

Thanks again!
// Jane

**Reply by Dominus98** [reply]
Hi, I just updated the code in Codepen with the solution you suggested

Thank you for the help!
